packages/web-runtime/themes/owncloud/theme.json

```diff
@@ -2,198 +2,201 @@
-          "swatch-brand-hover": "#223959",
-          "swatch-brand-contrast": "#ffffff",
-          "swatch-danger-contrast": "#ffffff",
-          "swatch-danger-default": "rgb(197, 48, 48)",
-          "swatch-danger-hover": "#b12b2b",
-          "swatch-danger-muted": "rgb(204, 117, 117)",
-          "swatch-inverse-default": "#ffffff",
-          "swatch-inverse-hover": "#ffffff",
-          "swatch-inverse-muted": "#bfbfbf",
-          "swatch-passive-default": "#4c5f79",
-          "swatch-passive-hover": "#43536b",
-          "swatch-passive-hover-outline": "#f7fafd",
-          "swatch-passive-muted": "#283e5d",
-          "swatch-passive-contrast": "#ffffff",
-          "swatch-primary-default": "#4a76ac",
-          "swatch-primary-hover": "#80a7d7",
-          "swatch-primary-muted": "#2c588e",
-          "swatch-primary-muted-hover": "rgb(36, 75, 119)",
-          "swatch-primary-gradient": "#4e85c8",
-          "swatch-primary-gradient-hover": "rgb(59, 118, 194)",
-          "swatch-primary-contrast": "#ffffff",
-          "swatch-success-default": "rgb(3, 84, 63)",
-          "swatch-success-hover": "#023b2c",
-          "swatch-success-muted": "rgb(83, 150, 10)",
-          "swatch-success-contrast": "#ffffff",
-          "swatch-warning-default": "rgb(183, 76, 27)",
-          "swatch-warning-hover": "#a04318",
-          "swatch-warning-muted": "rgba(183, 76, 27, .5)",
-          "swatch-warning-contrast": "#ffffff",
-          "text-default": "#041e42",
-          "text-inverse": "#ffffff",
-          "text-muted": "#4c5f79",
-          "icon-folder": "#4d7eaf",
-          "icon-archive": "#fbbe54",
-          "icon-image": "#ee6b3b",
-          "icon-spreadsheet": "#15c286",
-          "icon-document": "#3b44a6",
-          "icon-video": "#045459",
-          "icon-audio": "#700460",
-          "icon-presentation": "#ee6b3b",
-          "icon-pdf": "#ec0d47"
-        }
-      }
-    },
-    "default-dark": {
-      "general": {
-        "name": "ownCloud",
-        "slogan": "ownCloud – A safe home for all your data",
-        "privacyUrl": "",
-        "imprintUrl": ""
-      },
-      "appBanner": {},
-      "logo": {
-        "topbar": "themes/owncloud/assets/logo.svg",
-        "favicon": "themes/owncloud/assets/favicon.jpg",
-        "login": "themes/owncloud/assets/logo.svg",
-        "notFound": "themes/owncloud/assets/cloud.svg"
-      },
-      "loginPage": {
-        "autoRedirect": true,
-        "backgroundImg": "themes/owncloud/assets/loginBackground.jpg"
-      },
-      "designTokens": {
-        "breakpoints": {
-          "xsmall-max": "",
-          "small-default": "",
-          "small-max": "",
-          "medium-default": "",
-          "medium-max": "",
-          "large-default": "",
-          "large-max": "",
-          "xlarge": ""
+  "clients": {
+    "android": {},
+    "desktop": {},
+    "ios": {},
+    "web": {
+      "defaults": {
+        "appBanner": {
```

Comment:
Please kill the `appBanner` in the default theme. Was intentionally disabled for the time being (feature is not production ready).